packages/frontend/src/creation-form/index.tsx

```diff
@@ -162,12 +163,14 @@ export const Component = ({
             !validSpecification ||
             !state.timestamp ||
             !state.constraint ||
-            !state.constraint.type ||
+            state.constraint.type === undefined ||
             !state.constraint.value0 ||
             !state.constraint.value1 ||
```

Comment:
Is this ok?

Reply:
Yes, because the `constraint.type` could be `0` (`range`) so we need to explicitly check for `undefined`.

Reply:
No I meant the:

```
!state.constraint.value0 ||
!state.constraint.value1 ||
```

Reply:
Oh yes, I've fixed that in ddd881f356c0d996754a0e23c75539127cb3d07f.